.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```

Comment:
Should we change this to "Closes # (issue)"? I don't know whether or not "fixes" trigger GH to automatically link the issue in the same way that "closes" does.

Reply:
It does the same thing.
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
Perfect, thanks for confirming!